CMakeLists.txt

```diff
@@ -240,9 +240,19 @@ if(TRITON_BUILD_PYTHON_MODULE)
     MLIRIR
   )
 
-  set(ROCM_LIBRARIES
-    ${CMAKE_CURRENT_SOURCE_DIR}/lib/rocm/libhsa-runtime64.so
+  if(EXISTS "$ENV{ROCM_PATH}/lib/libhsa-runtime64.so.1" )
+  set(ROCM_LIBRARIES 
+    $ENV{ROCM_PATH/lib/libhsa-runtime64.so.1}
   )
+  elseif(EXISTS  "/opt/rocm/lib/libhsa-runtime64.so.1" )
+  set(ROCM_LIBRARIES 
+  /opt/rocm/lib/libhsa-runtime64.so.1
```

Comment:
I've been burnt in the past by using a hardcoded string that I thought was invariant, only to have to update all its occurrences at a later point. At least in this file and `lib/Target/HSACO/HSACOTranslation.cpp`, we should be able to use a CMAKE variable for the default value `/opt/rocm` and use it everywhere. And if cmake is invoked in the build by `python/setup.py`, it should be possible to encompass the hardcoded references in `python/triton/compiler.py` as well.